src/webots/nodes/WbRobot.hpp

```diff
@@ -141,6 +141,7 @@ public slots:
   void controllerChanged();
   void controllerRestarted();
   void controllerExited();
+  void robotResetted();
```

Comment:
This is not English, we should probably use `robotReset` or `robotWasReset` instead.

Reply:
Also, the use of the word robot is overkill as the class is already named Robot...